hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyLocationInfoGroup.java

```diff
@@ -177,15 +184,11 @@ void addAll(long versionToAdd, List<OmKeyLocationInfo> locationInfoList) {
 
   @Override
   public String toString() {
-    StringBuilder sb = new StringBuilder();
-    sb.append("version:").append(version).append(" ");
-    sb.append("isMultipartKey:").append(isMultipartKey).append(" ");
-    for (List<OmKeyLocationInfo> kliList : locationVersionMap.values()) {
-      for (OmKeyLocationInfo kli: kliList) {
-        sb.append(kli.getLocalID()).append(" || ");
-      }
-    }
-    return sb.toString();
+    return "OmKeyLocationInfoGroup{" +
```

Comment:
why the change? StringBuilder looks more elegant and performant.

Reply:
It was lacking detailed `locationVersionMap` output (no container ID, no bcsId). Also not formatted well so I am just using IntelliJ auto-generated `toString()` to replace it.

AFAIK using `StringBuilder` outside of a big loop should not make too much of a performance difference nowadays.

I might just restore it but add those interested fields.

Reply:
Done in 55b932fcd0432afc70e80f6a3b5868a276f5d0a0